nerdlets/landing/Home.js

```diff
@@ -137,6 +153,34 @@ class Home extends React.Component {
         <LayoutItem className="list-view">
           <Grid>
             <GridItem columnSpan={12}>
+              <Tile
+                to={navigation.getOpenStackedNerdletLocation({
+                  id: 'marketplace.home',
+                })}
+                type={Tile.TYPE.SOLID}
+                className="notice-tile"
+              >
+                <Badge
+                  type={Badge.TYPE.CRITICAL}
+                  spacingType={[
+                    Badge.SPACING_TYPE.OMIT,
+                    Badge.SPACING_TYPE.MEDIUM,
+                    Badge.SPACING_TYPE.OMIT,
+                    Badge.SPACING_TYPE.MEDIUM,
+                  ]}
+                  className="notice-badge"
+                >
+                  <Icon
+                    type={Icon.TYPE.INTERFACE__SIGN__EXCLAMATION__V_ALTERNATE}
+                    color="red"
+                  />{' '}
+                  Notice
+                </Badge>{' '}
+                We will be archiving this app on November 1st in favour of our
```

Comment:
just a slight rewording:

We will be archiving this application on November 1st 2021 and replacing  it with the [New Relic Instant Observability platform](add link to internal i/o).

Reply:
cool, the whole tile is actually clickable so should take them to internal IO

Reply:
👍